mln/services/friend.py

```diff
@@ -70,15 +70,9 @@ def handle_friend_invite_response(user, relation_id, accept):
 def remove_friend(user, relation_id):
 	"""
 	Remove a friend from the user's friend list.
-	Raise RuntimeError if
-	- the friendship does not exist
-	- the relation does not relate to this user, or

```

Comment:
These lines should still be kept, as they are a result of the `_get_friendship` call.

Reply:
Ah! Good catch.